tcp/sink_test.go

```diff
@@ -0,0 +1,63 @@
+package tcp
+
+import (
+	"context"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/rename-this/vhs/session"
+	"gotest.tools/v3/assert"
+)
+
+func TestNewSink(t *testing.T) {
+	cases := []struct {
+		desc        string
+		setup       func(*testing.T) (string, func())
+		errContains string
+	}{
+		{
+			desc: "bad addr",
+			setup: func(_ *testing.T) (string, func()) {
+				return "1.1.1.1:1111", func() {}
+			},
+			errContains: "i/o timeout",
+		},
+		{
+			desc: "bad addr",
```

Comment:
Should this test case have a unique description?

Reply:
Nice catch!